src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -2692,8 +2692,7 @@ public static void init() {}
                 System.setProperty("svnkit.http.methods","Digest,Basic,NTLM,Negotiate");
 
             // use SVN1.4 compatible workspace by default.
-            SVNAdminAreaFactory.setSelector(new SubversionWorkspaceSelector());
-        }
+            SVNAdminAreaFactory.setSelector(new SubversionWorkspaceSelector(workspaceFormat));
```

Comment:
This part is tricky. SVNKit apparently has static state, which is a bad idea generally (forces us to do some work on both the Jenkins controller & agent JVMs) and is especially weird since there is a user-configurable format which we have to somehow pass into static state. Hope test coverage is good enough here.